project_task_code/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2016 Tecnativa <vicent.cubells@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Sequential Code for Tasks",
+    "version": "15.0.0.0.1",
```

Comment:
Hi, thanks for your contribution.
I'm not sure that the version number follow the appropriate pattern.
Should be 15.0.1.0.1 IMHO.